app/models/talk.rb

```diff
@@ -143,4 +147,15 @@ def related_talks(limit: 6)
   def transcript
     enhanced_transcript.presence || raw_transcript
   end
+
+  def update_from_yml_metadata
```

Comment:
@adrienpoly should this be `update_from_yml_metadata!` or does the `performs` on L58 somehow figure that out by itself?

Reply:
indeed there is something wrong here will fix it on main thanks for the catch

Reply:
fixed https://github.com/adrienpoly/rubyvideo/commit/a1f1d5a8078e5a308425cfd31f9455ba63f05357